minimatch.js

```diff
@@ -709,7 +709,7 @@ function match (f, partial) {
   var options = this.options
 
   // windows: need to use /, not \
-  if (path.sep !== '/') {
+  if (!options.allowWindowsEscape && path.sep !== '/') {
```

Comment:
This should still be done.  Even if we aren't forcing `\` to be a path separator in the pattern, we still need to make the _path_ we're matching use `/` characters.